src/js/components/DataTable/stories/Custom.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, DataTable } from 'grommet';
+import { FormNext } from 'grommet-icons';
+
+// Source code for the data can be found here
+// https://github.com/grommet/grommet/blob/master/src/js/components/DataTable/stories/data.js
+import { columns, DATA } from './data';
+
+const customTheme = {
+  global: {
+    font: {
+      family: 'Helvetica',
+    },
+  },
+  dataTable: {
+    icons: {
+      sortable: FormNext,
```

Comment:
I personally found this icon to be confusing and too similar to the Next/Prev icons. can we use a different icon that will stand out?

Reply:
I scanned through grommet-icons looking for something that might work, to no avail. :(
Perhaps we should add an icon to grommet-icons?

Reply:
https://github.com/grommet/grommet-icons/issues/207

Reply:
In the meantime, I've built a custom icon into the story.

Reply:
And it looks great!